openedx/core/djangoapps/content_libraries/docs/decisions/0001-index-libraries-in-elasticsearch.rst

```diff
@@ -0,0 +1,57 @@
+1. Index libraries in elasticsearch
+-----------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The new content libraries reside in blockstore instead of edx-platform's models,
+which means that we are no longer able to query databases to get metadata
```

Comment:
Maybe say "to get **complete** metadata" because _some_ metadata is in the LMS database, and some in the Blockstore MySQL database.